homeworkweek3/src/components/laptops.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import Laptop from "./laptop";
+
+class Laptops extends Component {
+  constructor() {
+    super();
+    this.state = {
+      laptops: [
+        {
```

Comment:
Nu e gresit, insa cel mai ok e ca in constructor sa initializezi state-ul cu valori default, apoi aceste valori sa le setezi pe stare initial in ComponentDidMount. (se presupune ca astfel de date ar trebui sa vina de pe un server, call ce ar fi existat in componentDidMount). 

Locul de a schimba starea intr-un mod specific, e in ComponentDidMount, ComponentDidUpdate (de aceasta metoda vom discuta saptamana aceasta)

Eu as fi initializat astfel: this.state = { laptops: [] }